src/app/loginComponents/requests/requests.component.html

```diff
@@ -31,12 +31,11 @@ <h3>Curator Requests</h3>
           {{ org.topic }}
         </mat-card-content>
 
-        <mat-card-actions>
-          <div fxFlex></div>
-          <button mat-flat-button [id]="'reject-pending-org-' + i" class="mr-1" (click)="openDialog(org.displayName, org.id, false)">
+        <mat-card-actions fxLayout="row" fxLayoutAlign="end center" fxLayoutGap="10px">
+          <button mat-flat-button [id]="'reject-pending-org-' + i" class="mr-1" (click)="openDialog(org.displayName, org.id, false)" matTooltip="Reject an organization request">
             <mat-icon>clear</mat-icon>Reject
           </button>
-          <button mat-flat-button color="primary" [id]="'approve-pending-org-' + i" (click)="openDialog(org.displayName, org.id, true)">
+          <button mat-flat-button color="primary" [id]="'approve-pending-org-' + i" (click)="openDialog(org.displayName, org.id, true)" matTooltip="Approve an organization request">
```

Comment:
Just curious, why are you generating ids for here and elsewhere in the HTML? Is this because Angular can get confused if there is nothing unique distinguishing (I've seen that in AngularJS)?

Reply:
I am doing it for testing specifically.

Reply:
I generally follow https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements  , but I don't mind others using ID